sailor/pai/constants.py

```diff
@@ -0,0 +1,3 @@
+"""Please store all the constants here."""
+
+VIEW_ALERTS = '/alerts/odata/v1/Alerts'
```

Comment:
let's be more accurate.
`ALERTS_READ_PATH` would be better. 
we did not really come up with some good naming in the AC version.

Reply:
done